src/precice/tests/multi-coupling-three-solver.xml

```diff
@@ -0,0 +1,93 @@
+      <use-data name="DataAB" />
+      <use-data name="DataBA" />
+      <use-data name="DataCB" />
+      <use-data name="DataBC" />
+    </mesh>
+
+    <mesh name="MeshB1">
+      <use-data name="DataAB" />
+      <use-data name="DataBA" />
+    </mesh>
+
+    <mesh name="MeshB2">
+      <use-data name="DataBC" />
+      <use-data name="DataCB" />
+    </mesh>
+
+    <mesh name="MeshC">
+      <use-data name="DataBC" />
+      <use-data name="DataCB" />
+    </mesh>
+
+    <participant name="SolverA">
+      <use-mesh name="MeshA" provide="yes" />
+      <use-mesh name="MeshC" from="SolverC" />
+      <use-mesh name="MeshB1" from="SolverB" />
+      <use-mesh name="MeshB2" from="SolverB" />
+      <write-data name="DataAB" mesh="MeshA" />
+      <read-data name="DataBA" mesh="MeshA" />
+    </participant>
+
+    <participant name="SolverB">
+      <use-mesh name="MeshA" from="SolverA" />
+      <use-mesh name="MeshC" from="SolverC" />
+      <use-mesh name="MeshB1" provide="yes" />
+      <use-mesh name="MeshB2" provide="yes" />
+      
+      <write-data name="DataBA" mesh="MeshB1" />
+      <write-data name="DataBC" mesh="MeshB2" />
+      
+      <read-data name="DataAB" mesh="MeshB1" />
+      <read-data name="DataCB" mesh="MeshB2" />
+
+      <mapping:nearest-neighbor direction="read" from="MeshA" to="MeshB1" constraint="conservative" />
+      <mapping:nearest-neighbor direction="write" from="MeshB1" to="MeshA" constraint="consistent" />
+
+      <mapping:nearest-neighbor direction="read" from="MeshC" to="MeshB2" constraint="conservative" />
+      <mapping:nearest-neighbor direction="write" from="MeshB2" to="MeshC" constraint="consistent" />
+    </participant>
+
+    <participant name="SolverC">
+      <use-mesh name="MeshC" provide="yes" />
+      <use-mesh name="MeshA" from="SolverA" />
+      <write-data name="DataCB" mesh="MeshC" />
+      <read-data name="DataBC" mesh="MeshC" />
+    </participant>
+
+    <m2n:sockets from="SolverA" to="SolverB" />
+    <m2n:sockets from="SolverC" to="SolverA" />
+    <m2n:sockets from="SolverB" to="SolverC" />
+
+    <coupling-scheme:multi>
+      <participant name="SolverA" control="yes"/>
+      <participant name="SolverB" />
+      <participant name="SolverC" />
+      <max-time-windows value="10" />
+      <time-window-size value="1.0" />
+      <max-iterations value="2" />
+      <exchange data="DataBA" mesh="MeshA" from="SolverB" to="SolverA" />
+      <exchange data="DataAB" mesh="MeshA" from="SolverA" to="SolverB" />
+      <exchange data="DataCB" mesh="MeshC" from="SolverC" to="SolverB" />
+      <exchange data="DataBC" mesh="MeshC" from="SolverB" to="SolverC" />
+      <exchange data="DataCB" mesh="MeshC" from="SolverC" to="SolverA" />
+      <relative-convergence-measure data="DataCB" mesh="MeshC" limit="1e-4" />
+      <relative-convergence-measure data="DataBA" mesh="MeshA" limit="1e-4" />
+      <acceleration:IQN-ILS>
+        <data name="DataAB" mesh="MeshA" />
```

Comment:
We should add another testcase here, where all exchanged data is used in the acceleration (and maybe also a convergence measure exists for each of them).
... simply by iterating over configs in the test above (no additional code should be necessary).